<definition xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5" class="patch" id="oval:org.cisecurity:def:2303" version="6">
  <metadata>
    <title>DSA-3835-1 -- python-django -- security update</title>
    <affected family="unix">
      <platform>Debian 8</platform>
      <product>python-django</product>
    </affected>
    <reference ref_id="DSA-3835-1" ref_url="https://www.debian.org/security/dsa-3835" source="VENDOR" />
    <reference ref_id="CVE-2016-9013" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-9013" source="CVE" />
    <reference ref_id="CVE-2016-9014" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-9014" source="CVE" />
    <reference ref_id="CVE-2017-7233" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7233" source="CVE" />
    <reference ref_id="CVE-2017-7234" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7234" source="CVE" />
    <description>Several vulnerabilities were discovered in Django, a high-level Python web development framework.</description>
    <oval_repository>
      <dates>
        <submitted date="2017-04-28T10:47:46">
          <contributor organization="ALTX-SOFT">Sergey Artykhov</contributor>
        </submitted>
        <status_change date="2017-05-12T12:00:00.000-05:00">DRAFT</status_change>
        <status_change date="2017-05-26T16:17:25.745-04:00">INTERIM</status_change>
        <status_change date="2017-06-09T04:03:13.461-04:00">ACCEPTED</status_change>
      </dates>
      <status>ACCEPTED</status>
      <min_schema_version>5.10</min_schema_version>
    </oval_repository>
  </metadata>
  <criteria>
    <extend_definition comment="Debian 8 is installed" definition_ref="oval:org.mitre.oval:def:28919" />
    <criterion comment="python-django is earlier than 0:1.7.11-1+deb8u2" test_ref="oval:org.cisecurity:tst:3104" />
  </criteria>
</definition>
